defaults-o2.sh

```diff
@@ -53,6 +52,15 @@ overrides:
     tag: "v3.8.2"
     prefer_system_check: |
       which cmake && case `cmake --version | sed -e 's/.* //' | cut -d. -f1,2,3 | head -n1` in [0-2]*|3.[0-7].*) exit 1 ;; esac
+  simulation:
```

Comment:
Please modify directly `simulation.sh`. We use it only for O2, no need to override it in the defaults.

Reply:
ok, I modified as requested, please see updated PR.

It was although not clear to me that simulation.sh package is only used in O2, according to the structure. I mean, why then the dependencies tracked in simulation.sh are not explicitly included in the o2.sh receipt? That looked to me like someone was using this as a package collection.

Anyway, the change is done.